SU2_CFD/include/variables/CVariable.hpp

```diff
@@ -137,6 +137,20 @@ class CVariable {
     RegisterContainer(input, variable, &ad_index);
   }
 
+  void RegisterContainer(bool input, su2activevector& variable, su2vector<int>* ad_index = nullptr) {
```

Comment:
I don't like this almost-duplicate but I couldn't decide on the best alternative, too.

Reply:
Templating the other one didn't work?